### ISSUE TITLE
Visually highlight more relevant (based on filter/search) markers on the map view

### ISSUE THREAD

**Issue by macsyyyyyyy** [androidseb/mapmarker#42]
To date, the same point of interest can only be added to one folder (map). This is very limiting for the application. Would it be possible for you to improve this as indicated below?

When a point of interest is contained in two (or more) folders (maps) among those displayed

it means that that point of interest meets both conditions of what I am looking for

so it would be very helpful if its marker would be larger so that it stands out well : that point of interest is exactly what I'm looking for!

**Comment by androidseb** [comment]
The description you're making of folders seems similar to "tags" or "labels" (feature not built at this point in time) that we would use to categorize POIs.

A POI can only be in a single folder, however there could be multiple tags per POI.

In the current state of the app, you can duplicate a POI across multiple folders but it will make your map messy because edits will have to be done manually in several places when you make changes to the POI, so it's probably not a UX I'd be going towards...

However this idea seems interesting down the line once a couple of yet-to-be-built features will be around:

* "tags" or "labels" for POIs, with ability to search POIs by tags
* Display search results on the map

At that point (future), in order to display marker search results with the most relevant matches (matching name name, number of matching tags, etc.) on the map, it could be interesting to make them vary in size.

@macsyyyyyyy what are your thoughts on this?

**Comment by macsyyyyyyy** [comment]
I agree with you. I generally hate folders and love Tags. Folders hide, Tags highlight. 

When you show several folders on the same map, implicitly you are using the folders as if they were tags. This is the real strength of the app compared to google maps, which in contrast does not allow you to show different listings within the same map.

**Comment by androidseb** [comment]
I understand you're using folders as labels because there is nothing else...

I've created a dedicated issue for adding the labels feature because it was not on the Backlog yet:
https://github.com/androidseb/mapmarker/issues/44

I think that completing that will be a pre-requirement to considering what you're asking for in this issue.

The app already supports setting the size of markers on the map in the settings, that will conflict a little bit with this idea of search results with smaller/bigger markers. I wonder if there couldn't be a better way to visually distinguish search results from one another in relevance, maybe by playing with the opacity of markers?

**Comment by macsyyyyyyy** [comment]
Yes, it's true. When you show me a folder I see a label, because in general I hate folders :D 

I think opacity is not enough, so as not to mess up the map and avoid layout effects that might work in one case and not work in another

in my opinion, a robust and effective solution would be to reserve a wildcard shape and/or color (user's choice) to highlight these markers that satisfy different tags and also to add an optional filter into the left panel that allows to show on the map only the markers that satisfy 2 or more tags among chosen

what do you think?

**Comment by androidseb** [comment]
I think that the question "how to achieve this" will become easier to think about once tags are implemented and that UX is finalized.

So for now I'll mark this as "future improvement" because in the current state of the app I don't think we can find an actionable good response for this.

We could clarify the title however, I was thinking of renaming this to "Visually highlight more relevant (based on filter/search) markers on the map view"?

**Comment by macsyyyyyyy** [comment]
Yes, i agree with you
